src/Adapters/Auth/phantauth.js

```diff
@@ -0,0 +1,46 @@
+/*
+ * Helper functions for authenticating with PhantAuth.
+ *
+ * PhantAuth was designed to simplify testing for applications using OpenID Connect
+ * authentication by making use of random generated users.
+ *
+ * To learn more, please go to: https://wwww.phantauth.net
+ */
+
+var Parse = require('parse/node').Parse;
+const httpsRequest = require('./httpsRequest');
+
+// Returns a promise that fulfills if this user id is valid.
+function validateAuthData(authData) {
+  return request('auth/userinfo', authData.access_token).then(data => {
+    if (data && data.sub == authData.id) {
+      return;
+    }
+    throw new Parse.Error(
```

Comment:
Can you add a test case for success and error? There should be examples in `spec/AuthenticationAdapters.spec.js`

Reply:
I already added test entry to "responses" mock array. As I see other provider do it same.
Do you mean I should create separate "describe" for phantauth insted of using comon one?

Reply:
Yes